### PR TITLE
Play point animation when sending a follower into battle

### PR DESCRIPTION
**Pull request by kittenchilly**
## Description
Merrp's branch had a custom point animation, but the changes for it to actually play weren't merged properly, so this fixes that
## **Discord contact info**
kittenchilly

### CODE REVIEW

**Comment by kittenchilly** [comment]
Closing in favor of PRing merrp's branch instead

**Comment by kittenchilly** [comment]
Reopening actually since apparently this is a bug in expansion, it works fine in merrp's branch!

File: src/data/graphics/trainers.h [hunk 1]
```diff
@@ -425,21 +425,17 @@ static const union AnimCmd sAnimCmd_Kanto[] =
...
 static const union AnimCmd sAnimCmd_Point_HGSS_Red_Leaf[] =
 {
-    ANIMCMD_FRAME(0, 9),
-    ANIMCMD_FRAME(1, 9),
-    ANIMCMD_FRAME(3, 24),
-    ANIMCMD_FRAME(3, 24),
-    ANIMCMD_FRAME(0, 50),
+    ANIMCMD_FRAME(1, 31),
+    ANIMCMD_FRAME(4, 24),
+    ANIMCMD_FRAME(0, 1),
```
Comment (Bassoonian) [inline src/data/graphics/trainers.h:438]: I get that the other change was a merge error on expansion's part, but are these altered cmd frames also a merge error or are these your own changes?

Comment (kittenchilly) [reply src/data/graphics/trainers.h:438]: These were my own changes. I felt the one there already felt too delayed between the start of the point and the actual slide in of the mon, so I changed it to make it feel more like HGSS.

Comment (Bassoonian) [reply src/data/graphics/trainers.h:438]: Then I personally believe that we should follow merrp's branch, with these suggestions going to her branch and merrp being allowed to decide. The fix itself can still go to expansion, though.

Comment (kittenchilly) [reply src/data/graphics/trainers.h:438]: Fair enough, I'll move these changes

Comment (kittenchilly) [reply src/data/graphics/trainers.h]: Done

**Review by Bassoonian — APPROVED** [review]

**Pull request merged**